### ISSUE TITLE
Change Ines' name on Companies House, HMRC & banks

### ISSUE THREAD

**Issue by iteles** [dwyl/hq#493]
My IDs are expiring at the end of August and my new ones have my married name. 💍 

As such, we need to change my name with Companies House, HMRC, banks and any other legal documents where I may at some point need to prove my identity and not have any documents to do so (my marriage license is in Portuguese, so it would be a long and bureaucratic process if it needed to be used).

I usually just use first and last so `Ines Teles` ➡️ `Ines Teles Correia`
Where my full name is used, we need to _add_ `Correia` on the end.

* [x] Companies house - submitted 9/10
* ~HMRC~ - we have a `dwyl` user account not registered to a person
* [x] Coop bank
* [ ] Santander Bank
* Other (specify)

**Comment by iteles** [comment]
@rub1e This can happily wait until next week and should not derail this one 👍

**Comment by rub1e** [comment]
@iteles for the Coop, you need to do the following:

* Send a letter to 

Cooperative Bank
PO Box 250
Delf House
Skelmersdale
WN8 6WYT

* Letter to contain
  * Company Name
  * S/C & Account number
  * Current name and desired change
  * Copy of old signature
  * Copy of new signature
  * *Original* marriage certificate OR *certified* copy
  * Note with your contact details saying you'd like to get confirmation when the change is made

Unfortunately I can't help you much with the above, given our non-co-location 😞

**Comment by rub1e** [comment]
@iteles Are you visiting the UK soon? Just spoke to Santander and they're a bit iffy with changing names by post - we'd need to remove you from the account and then re-add you under your new name, and I don't really trust them not to screw that up.

So the easiest thing to do is for you to go into a branch and they can process it for you (it's the only time they seem to be willing to do anything in-branch for biz customers).

Basically if you go into a branch with all the info need for the Coop change, they'll sort you out. 

Failing that, I can try the `Change of Details` form, but the guy on the phone didn't sound too confident that would work (sounds like a hack tbh)
